Userland/DevTools/SQLStudio/MainWidget.cpp

```diff
@@ -135,6 +142,14 @@ MainWidget::MainWidget()
         update_editor_actions(editor);
     });
 
+    m_run_script_action = GUI::Action::create("Run script", { Mod_Alt, Key_F9 }, Gfx::Bitmap::try_load_from_file("/res/icons/16x16/program-run.png").release_value_but_fixme_should_propagate_errors(), [&](auto&) {
```

Comment:
I didn't mean using the Hack Studio icon for this, which only makes sense next to the others (see screenshot) - just use the [play icon](https://github.com/SerenityOS/serenity/blob/master/Base/res/icons/16x16/play.png) please, as suggested.

![image](https://user-images.githubusercontent.com/19366641/174852124-fb4c4fda-be09-47ae-9857-0724c40df7ac.png)